package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "js-marketing-notification",
-  "version": "1.2.1",
+  "version": "1.3.1",
   "description": "A module for displaying tasteful marketing content to users upon first visit to page.",
-  "main": "src/js-marketing-notification.js",
+  "main": "js-marketing-notification.ts",
```

Comment:
@Izzmo using the `.ts` file will cause any of our GitHub repos to go through local TS compiler as well. Just stating to see if that is something we are wanting for all going forward.